### ISSUE TITLE
Dead :(

### ISSUE THREAD

**Issue by elisaado** [wenchaojiang/ev3-Nodejs-bluetooth-Api#8]
I use this "api" almost every time when I use my EV3 brick, and I've actually built something cool with it, though, it seems like the project is dead, which is pretty awful. The only extra function I need is the ultrasone sensor, haven't looked at the code yet, maybe I can add it myself, but It'd be very nice if this "api" was complete, I love it!

**Comment by wenchaojiang** [comment]
yes, I personally does not have that device. I am looking forward to your
contribution
